index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Mobile Version Skeletun</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav>
+    <ul class="bar">
+      <li>John Doe</li>
```

Comment:
- [ ] The logo **John Doe** should be a link (`a` tag). This helps users easily click and access the main idea of what you page is all about from your logo